pages/common/smalltalkci.md

```diff
@@ -0,0 +1,28 @@
+# smalltalkci
+
+> Framework for testing Smalltalk projects with GitHub Actions, Travis CI, AppVeyor, GitLab CI, and others.
+> More information: <https://github.com/hpi-swa/smalltalkCI#readme>
+
+- Run tests for a config file:
+
+`smalltalkci {{/path/to/.smalltalk.ston}}`
+
+- Run tests for the `.smalltalk.ston` configuration in the current directory:
+
+`smalltalkci`
+
+- Debug tests in headful mode (show VM window):
+
+`smalltalkci --headful`
+
+- Specify prepared [S]malltalk image:

```

Comment:
I don't understand exactly what this example does.

Reply:
If `smalltalkci` is passed the `--smalltalk` option, it downloads and prepares a Smalltalk image for the given Smalltalk distribution and version. For instance, possible arguments would be:

```sh
smalltalkci --smalltalk Squeak64-Trunk  # bleeding edge Squeak image for 64-bit VMs
smalltalkci --smalltalk Squeak32-5.3  # Squeak version 5.3 image for 32-bit VMs
smalltalkci --smalltalk Pharo64-stable  # Pharo stable image for 64-bit VMs
```

This is described here in greater detail: <https://github.com/hpi-swa/smalltalkCI#images>

Hmm... Maybe change the description to this?

```suggestion
- Download and prepare a well-known [s]malltalk image for the tests:
```

Reply:
Yeah, sounds better, you can remove the `[]` because now you' re using the long argument format.